config/owasp/suppressions.xml

```diff
@@ -31,6 +32,7 @@
     <cve>CVE-2022-40152</cve>
     <cve>CVE-2022-46363</cve>
     <cve>CVE-2022-46364</cve>
+    <cve>CVE-2022-40152</cve>
```

Comment:
Remove this duplicate CVE declaration from both here and line 24 in the notes section.